### PR TITLE
Feature/63 admin ux tweaks

### PR DESCRIPTION
**Pull request by finnlewis**

### CODE REVIEW

File: modules/localgov_services_page/config/install/field.field.node.localgov_services_page.localgov_services_parent.yml [hunk 1]
```diff
@@ -6,15 +6,15 @@ dependencies:
     - node.type.localgov_services_landing
     - node.type.localgov_services_page
     - node.type.localgov_services_sublanding
-  module:
-    - localgov_services_navigation
+_core:
+  default_config_hash: 8pYEb1Mvn2qzDXMlwm1II8CRTlryGkwUMoo90KHzbas
```
Comment (ekes) [inline modules/localgov_services_page/config/install/field.field.node.localgov_services_page.localgov_services_parent.yml:10]: Don't want this I think. But probably do want the dependency on localgov_services_navigation?

File: modules/localgov_services_page/config/install/core.entity_form_display.node.localgov_services_page.default.yml [hunk 2]
```diff
@@ -2,15 +2,16 @@ langcode: en
 status: true
 dependencies:
   config:
+    - entity_browser.browser.page_components
```
Comment (ekes) [inline modules/localgov_services_page/config/install/core.entity_form_display.node.localgov_services_page.default.yml:5]: I guess this is a paragraphs requirement that isn't strictly related to this commit?

File: modules/localgov_services_page/config/install/core.entity_form_display.node.localgov_services_page.default.yml [hunk 3]
```diff
@@ -67,28 +69,30 @@ third_party_settings:
...
+      weight: 6
       format_type: tabs
       format_settings:
         id: ''
         classes: ''
         direction: vertical
       label: tabs
       region: content
+_core:
+  default_config_hash: tpnylkwL_ZARGkzfR6fWGa5mI-QEdisVDe9Da0pCZLg
```
Comment (ekes) [inline modules/localgov_services_page/config/install/core.entity_form_display.node.localgov_services_page.default.yml:88]: Ach. Another one.

**Comment by ekes** [comment]
Just some commit clean up I think.

**Comment by finnlewis** [comment]
I need to fix the unmet dependencies on teh config.

**Pull request merged**